postem/postem-app/src/main/webapp/WEB-INF/templates/create_gradebook.html

```diff
@@ -64,11 +64,11 @@ <h1 th:text="#{create_update}" /></h1>
                     <label for="gradeBookFeedBackAvail" class="form-control-label"
                         th:text="#{gradebook_feedbackavail}">Feedback Availability</label>
                 </div>
-                <div class="col-sm-4">
-                    <input type="checkbox" name="gradeBookFeedBackAvail" id="released"
-                        th:field="*{released}" /> <label for="gradeBookFeedBackAvail"
-                        class="form-control-label" th:text="#{release}">Release
-                        feedback to participants?</label>
+                <div class="col-sm-4 text-nowrap">
+                    <input type="checkbox" name="gradeBookFeedBackAvail" id="released" th:field="*{released}" />
+                    <label for="released" class="form-control-label" th:text="#{release}">
+                        Release feedback to participants?
```

Comment:
i18n?

Reply:
Not needed. Thymeleaf replace the tag contents. It's just a placeholder.

Reply:
th:text does the magic.

Reply:
Great!. Thanks